### PR TITLE
kubernetes-setup Dockerfile

### PR DESCRIPTION
**Pull request by pmalek-sumo**
###### Description

Fill in your description here.

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

**Review by astencel-sumo — APPROVED** [review]
I'm not in love with the `setupjob` single word 🤪
How about `setup-job` or just `setup`?

**Comment by perk-sumo** [comment]
> How about setup-job or just setup?

I'd go with simple `setup`.

File: deploy/docker/setupjob/Makefile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+BUILD_TAG ?= latest
+IMAGE_NAME = setupjob
```
Comment (perk-sumo) [inline deploy/docker/setupjob/Makefile:2]: This should rather be a full name like: `sumologic-kubernetes-setup` or sth.

Comment (pmalek-sumo) [reply deploy/docker/setupjob/Makefile]: Renamed it to `kubernetes-setup` as the alias we're going to get will be `sumologic` (hopefully :) )

**Review by astencel-sumo — APPROVED** [review]
👏

**Pull request merged**